src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -412,11 +412,12 @@ internal interface ISyntaxFactsService : ILanguageService
 
         ImmutableArray<SyntaxNode> GetSelectedFieldsAndProperties(SyntaxNode root, TextSpan textSpan, bool allowPartialSelection);
         bool IsOnTypeHeader(SyntaxNode root, int position);
-        bool IsInHeader(SyntaxNode node, SyntaxNode ownerOfHeader, SyntaxNodeOrToken lastTokenOrNodeOfHeader);
-        bool IsInPropertyDeclarationHeader(SyntaxNode node);
-        bool IsInParameterHeader(SyntaxNode node);
-        bool IsInMethodHeader(SyntaxNode node);
-        bool IsInLocalFunctionHeader(SyntaxNode node);
+        bool IsOnHeader(int position, SyntaxNode ownerOfHeader, SyntaxNodeOrToken lastTokenOrNodeOfHeader);
```

Comment:
would still like this particular method to not be exposed on the interface itself.

Reply:
I'll see if I can get rid of it, IIRC there was some issue with it needing to be not sure.